app/uk/gov/hmrc/cataloguefrontend/MetricsExplorerController.scala

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cataloguefrontend
```

Comment:
Not something that is done consistently in catalogue-frontend, but it would be good to use a feature package. See `healthindicators`, `shuttering` and `whatsrunningwhere`.
Especially because the model has global sounding names, but look like they only apply to this feature.

Also consider putting the model direct into your new `metrics` package - they are cross cutting across the feature, not just for the `connector` (and also having a `model` package doesn't make sense to me from a package semantic) - appreciate this is not how a lot of the code has been done in catalogue already!

Reply:
agreed. have done as you suggested

Reply:
I was expecting `MetricsExplorerController` to also move to the `metrics` package.
I don't think we need or want the `connector` or `model` sub-packages (to use an API the client shouldn't have to import a separate model package). 
I'd be tempted to put all the model in to a single `model.scala` file - a file for each type doesn't pull it's weight.

Reply:
thought it's helpful for folks to find all their controllers in the place they expect them to. also was having problems with `action =  appRoutes.MetricsExplorerController.search` in the template as the generated appRoutes wouldn't have the `MetricsExplorerController` entry. 

as for whether or not we should have separate sub-packages, or having models in a single file, or having separate files, i think that's all a bit personal, and not worth fighting over. :)